sdk/resourcemanager/Azure.ResourceManager.Core/docs/MigratingFromCurrentToPreview.md

```diff
@@ -0,0 +1,86 @@
+# Differences between current and preview .NET Management SDK
```

Comment:
Why do we have such a prominent doc for migrating from previous track 2 management sdks to the new ones?  This is a tiny fraction of management sdk usage.  The more important scenario is migrating from *track 1* management sdks.